src/renderer/ui/components/modals/GoToModal.js

```diff
@@ -54,10 +81,18 @@ export default class GoToModal extends Component {
       remote.app.quit();
     } else if (url === 'DEBUG_INFO') {
       Emitter.fire('generateDebugInfo');
+      this.handleClose();
+    } else {
```

Comment:
Should we then verify it again, after resolution? I assume yes, so I should refactor out the test into its own function.